src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -244,7 +244,10 @@
                                              first
                                              key)
          ownership-status            (get ownership-status-by-address owner-address)
-         collectibles                (data-store/rpc->collectibles collectibles)
+         collectibles                (->> collectibles
+                                          data-store/rpc->collectibles
+                                          (map #(update % :ownership distinct))
+                                          vec)
```

Comment:
was the problem due to ownerships being duplicated? :thinking: 

Any reason behind why we receive such information from status-go?

Reply:
Yes, partially it was.
Don't know the reason, I tried to investigate it in `status-go` but that took too much time, so I decided to fix it on mobile side first.